File: java/src/main/java/toolgood/flowVision/Flows/FactoryWork.java
package toolgood.flowVision.Flows;

import com.alibaba.fastjson2.JSONObject;

public class FactoryWork {
    public String Code;
    public String Name;

    final static FactoryWork parse(JSONObject jsonObject) {
        FactoryWork result = new FactoryWork();
        result.Code = jsonObject.getString("code");
        result.Name = jsonObject.getString("name");

        return result;
    }

    public void Init(ProjectWork work) {
    }
}

File: csharp/ToolGood.FlowVision/Flows/FactoryWork.cs
using System.Text.Json.Serialization;

namespace ToolGood.FlowVision.Flows
{
	public sealed class FactoryWork
	{
		/// <summary>
		/// 编号
		/// </summary>
		[System.Text.Json.Serialization.JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
		public string Code { get; set; }

		/// <summary>
		/// 简化名称
		/// </summary>
		[System.Text.Json.Serialization.JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
		public string Name { get; set; }

		internal void Init(ProjectWork work)
		{
		}
	}
}